exdb/views.py

```diff
@@ -51,9 +51,11 @@ def get_hs_queryset(self):
         experience_approvals = ExperienceApproval.objects.filter(
             approver=self.request.user, experience__status='ad'
         )
+        next_approver_queue = Q(next_approver=self.request.user) & ~Q(status="dr")
```

Comment:
My original thought was to actually change the final `return` statement to:
```python
return experiences | self.get_ra_queryset()
```
Which should theoretically merge the two queries... but would take some testing to be sure everything worked properly. To answer your question: yeah, it should probably only show pending experiences where they're the next_approver.